app/src/main/java/space/taran/arknavigator/ui/fragments/dialog/EditTagsDialogFragment.kt

```diff
@@ -94,5 +98,23 @@ class EditTagsDialogFragment(resourceId: ResourceId, storage: TagsStorage, index
         binding.etNewTags.setText("")
     }
 
+    override fun dismissDialog() {
+        dismiss()
+    }
+
     override fun getTheme() = R.style.FullScreenDialog
+
+    companion object {
+        const val TAG = "editTagsDialogTag"
```

Comment:
Let's call it `FRAGMENT_TAG` or something like this, to not confuse it with our tags.